imagersite/imagerprofile/models.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class ActiveProfileManager(models.Manager):
```

Comment:
It'd be good to have a class docstring that documented the purpose of this custom model manager.